lib/tasks/redirect_legacy_email_signups.rake

```diff
@@ -0,0 +1,29 @@
+desc "Redirect legacy email signups"
+task redirect_legacy_email_signups: :environment do
+  # Exclude parent specialist sectors as they never included an email signup option
+  specialist_sectors_info = Topic.where(state: "published").where.not(parent_id: nil).map do |ss|
+    { state: ss.state, slug: ss.full_slug }
+  end
```

Comment:
You don't need to pull out state any more. You can do `Topic.where(state: "published").where.not(parent_id: nil).pluck(:full_slug)` (presuming that is a db column rather than an instance method)

Reply:
Doh of course, sloppy half job removal of this code! 😆